### ISSUE TITLE
Staging Review finding: Unnecessary text for screen reader users on power of attorney requests

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104053]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/representative/poa-requests](https://staging.va.gov/representative/poa-requests?status=pending&sort=created_at_asc), there are a couple of instances where `aria-label` and `sr-only` text has been added to an element that isn't necessary:
- The pending and processed tabs have `aria-label="View requests marked as [pending/processed]"`
- For each card, the link includes `<span class="sr-only">View details for</span>` before the Veteran's name

In each case, the extra verbosity isn't really necessary, and the repetition may actually frustrate screen reader users trying to move through the page quickly.

It also introduces a potential barrier for voice command / speech recognition software users. When a voice user speaks a command like "Click Pending," their software looks for an element on the page whose name matches what the user said. But some voice software matches against the what's in the browser's accessibility tree rather than the text that's visible --- which means looking at the `aria-label` or other screen reader text for matches. Some more sophisticated voice software will still be able to get a match from "Pending" to "View requests marked as pending," but other voice software fails if there's not a more exact match.

Voice software users have other ways of interacting with elements, so if the extra text was offering a clear benefit to screen reader users it could be justifiable. But since it's not really providing any extra value to screen reader users, I don't think it's worth introducing the extra barrier.

<details><summary>Link, screenshot or steps to recreate</summary>

Pending/Processed tabs currently coded as:
```
<a class="poa-request__tab-link active" role="tab" id="tab-pending" aria-controls="tabpanel-pending" aria-selected="true" aria-label="View requests marked as pending" href="/representative/poa-requests?status=pending&amp;sort=created_at_asc">Pending</a>
```

Card links currently coded as:
```
<a href="...">
   <span class="sr-only">View details for </span>
   <h3 data-testid="..." class="poa-request__card-title vads-u-font-size--h4">Veteran, Test</h3>
</a>
```

</details>

## Recommended action

Remove the `aria-label` from the tabs, and remove the `sr-only` text from the links for each card.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jquispe-oddball** [comment]
submitted pr 
https://github.com/department-of-veterans-affairs/vets-website/pull/35023

**Comment by jquispe-oddball** [comment]
Removed aria label from tabs, sr text from cards, see screenshots. Ticket can be closed @candilemoine 

<img width="454" alt="Image" src="https://github.com/user-attachments/assets/6ae18174-9156-45a0-bab0-77034f04106e" />
<img width="458" alt="Image" src="https://github.com/user-attachments/assets/401623b7-9f44-4c42-bc71-c36e81a987ca" />
